project/__init__.py

```diff
@@ -20,12 +22,13 @@
 
 def cronjob():
     cron = CronTab()
-    job = cron.new(command = 'curl http://127.0.0.1:5000/api/reminder', comment = "Reminding all users of jobs to be completed today.")
+    job = cron.new(command = 'curl http://127.0.0.1:5000/api/reminder', 
```

Comment:
Can you make  127.0.0.1:5000 this a global variable and use that variable in command

It will be easier to change the address in that variable